services/mozilla-observatory/mozilla-observatory.service.tester.js

```diff
@@ -0,0 +1,224 @@
+'use strict'
+
+const { ServiceTester } = require('..')
+
+const t = (module.exports = new ServiceTester({
+  id: 'mozilla-observatory',
+  title: 'Mozilla Observatory Scanner',
+}))
+
+t.create('request on httpforever')
+  .get('/httpforever.com.json?style=_shields_test')
+  .expectJSON({
+    name: 'observatory',
+    value: 'C+ (60/100)',
+    color: 'aa7039',
+  })
+
+t.create('grade A (mock)')
+  .get('/foo.bar.json?style=_shields_test')
+  .intercept(nock =>
+    nock('https://http-observatory.security.mozilla.org')
+      .get('/api/v1/analyze?host=foo.bar')
+      .reply(200, { grade: 'A', score: 115 })
+  )
+  .expectJSON({
+    name: 'observatory',
+    value: 'A (115/100)',
+    color: '2d882d',
```

Comment:
The `color` values in the response will have the `#` prefix so those will need to be added for the tests to pass

Reply:
I don't get it... I've added `#` but now the tests are failing 🤔 

https://circleci.com/gh/badges/shields/40108

```
AssertionError: expected { Object (color, name, ...) } to deeply equal { Object (name, value, ...) }
```

Reply:
I'm now using Shields colors anyway